packages/moertel/src/mortar/mrtr_overlap_Def.hpp

```diff
@@ -381,8 +381,37 @@ bool MOERTEL::Overlap<IFace>::build_sxim()
   double gap;
   for (int i=0; i<nsnode; ++i)
   {
+
+    {
+      const double * node_norm = snode[i]->Normal();
+
+      double master_normal[3];
+      MOERTEL::Node** nodes = mseg_.Nodes();
+      for (std::size_t i=0; i < mseg_.Nnode(); ++i)
+      {
+        const double * node_norm_tmp = nodes[i]->Normal();
+        master_normal[0] += node_norm_tmp[0];
+        master_normal[1] += node_norm_tmp[1];
+        master_normal[2] += node_norm_tmp[2];
+      }
+
+      double norm1 = sqrt(MOERTEL::dot(master_normal,master_normal,3));
+      master_normal[0] /= norm1;
+      master_normal[1] /= norm1;
+      master_normal[2] /= norm1;
+
+      double mag_projection = fabs(MOERTEL::dot(node_norm,master_normal,3));
+
+      if(mag_projection <= Projection_Length_Epsilon)
```

Comment:
This could print on every MPI process.  Users need some way to turn off these messages.

Reply:
> @kliegeois The only thing I'm worried about is all the printing to `stdout` on every MPI process. Otherwise, I'm totally OK with this, but I'd rather hear from an actual Moertel developer. Are there any left, or are you the only one now?

Well, those printing can be removed without any issue I think.
I do not know if anyone is working on Moertel right now.